yahoo_panoptes/framework/plugins/runner.py

```diff
@@ -188,7 +188,7 @@ def execute_plugin(self):
             self.warn(plugin, 'Plugin did not return any results')
             return
         elif not isinstance(results, self._plugin_result_class):
-            logger.warn(plugin, 'Plugin returned an unexpected result type: "{}"'.format(type(results).__name__))
+            self.warn(plugin, 'Plugin returned an unexpected result type: "%s"' % type(results).__name__)
```

Comment:
Could we please change all the formatting to the old style?
Also, could you please move logger.info on line 222 to self.info